packages/analytics-browser/src/utils/web-attribution-helper.ts

```diff
@@ -0,0 +1,88 @@
+import { BASE_CAMPAIGN } from '@amplitude/analytics-client-common';
```

Comment:
Curious: Why is this a separate file and not a part of the web-attribution.ts file?

Reply:
- Modify anything in the web attribution plugin will release a new version while we publish, this duplicate will not. 
- There is a case for using web attribution separately in browser 1.0. Copy over is for isolating purposes.  
Let me what do you think.

Reply:
Why is it a concern that modifying the web attribution plugin will release a new version?

Copy over is fine but we don't want code to be duplicated in two places right? We can still keep the web attribution plugin and just call these util methods right?

Reply:
Make sense. 
I have put the web attribution logic in analytics-client-common.

Reply:
I think Justin had a different suggestion. Please check that as well.

Reply:
I prefer my current way, instead of putting the util in the web attribution, because it can prevent installing the web attribution package since the plugin stuff is irrelevant. 
We already have something in the analytics-client-common for the attribution, put the web attribution logic in analytics-client-common and import the logic in web attribution plugin and browser-client can also prevent duplication code. 

@justin-fiedler Let me know what you think.

Reply:
https://github.com/amplitude/Amplitude-TypeScript/pull/696#discussion_r1560102752